biogtr/training/train.py

```diff
@@ -27,26 +34,45 @@ def main(cfg: DictConfig):
     """Main function for training.
 
     Handles all config parsing and initialization then calls `trainer.train()`.
+    If `batch_config` is included then run will be assumed to be a batch job.
 
     Args:
         cfg: The config dict parsed by `hydra`
     """
     train_cfg = Config(cfg)
-
+    pprint(f"Base train config: {train_cfg.cfg}")
+
+    # update with parameters for batch train job
+    if "batch_config" in cfg.keys():
+        try:
+            index = int(os.environ["POD_INDEX"])
```

Comment:
This assumes the hparams csv is already created - we should also add ability to create the csv given desired grid search hparams.

Reply:
TODO: Come back to this